cmd/operator/main.go

```diff
@@ -119,19 +117,15 @@ func main() {
 }
 
 func runCLI(logger *logrus.Logger, cfg *config.Config) {
-
-	runclient, err := run.NewAPIClient(context.Background(), flRegions[0])
-	if err != nil {
-		logger.Fatalf("could not initilize Cloud Run client: %v", err)
-	}
-	roll := rollout.New(&service.Client{
-		RunClient:   runclient,
-		Project:     cfg.Targets[0].Project,
-		ServiceName: "hello",
-		Region:      flRegions[0],
-	}, cfg.Strategy).WithLogger(logger)
-
 	for {
+		services := service.Filter(logger, cfg)
+		if len(services) == 0 {
+			logger.Info("No service matches the targets")
+			interval := time.Duration(cfg.Strategy.Interval)
+			time.Sleep(interval * time.Second)
+		}
+		roll := rollout.New(services[0], cfg.Strategy).WithLogger(logger)
```

Comment:
This is definitively a TODO. I didn't want to overcomplicate this PR since rollout of several services would be in another PR. I'll add a TODO here. Thanks!